### PR TITLE
Solarmoth and Lunarmoth port ^2

### PR DESCRIPTION
**Pull request by Saisni**
## About The Pull Request

Ports Solarmoth and Lunarmoth simplemobs as an evolution of juvenile solargrubs if left alive for quite a long time. Yes, if a grub is left to feast on power - it will eventually turn either into a Solarmoth or a Lunarmoth. They are slightly more tame versions of VIRGO ones, with a limit on temperature buildup, from Chompstation;

## Why It's Good For The Game

Grubs are currently just seek/destroy type of event with no rhyme or reason. With solar and lunar grubs you get a bit more spice to the event and adds a few options for ones clevermind to abu...exploit the poor butterflies.

RvafILjfZw
Vtdg9hlYll
zOIW9hSMTD

## Changelog
:cl:
add: Solarmoths and lunarmoths simplemobs
tweak: Grub code is tweaked to spawn moths
imageadd: Solarmoth icons are added
spellcheck: Capitalization
code: Added code for the solar and lunar moths.
/:cl:

### CODE REVIEW

**Comment by Saisni** [comment]
I have no idea why Linters is being a bitch, because everything's working this way on this most recent pull.

**Comment by Saisni** [comment]
Ok, I'm just a dumbass; 
Remember that ticking your DMs doesn't automatically add them to the core DME! 

Thanks CharlesWedge!

**Review by silicons — APPROVED** [review]
this has my approval

File: code/modules/mob/living/simple_mob/subtypes/vore/solargrub.dm [hunk 1]
```diff
@@ -18,16 +18,24 @@ GLOBAL_LIST_EMPTY(solargrubs)
 	value = CATALOGUER_REWARD_EASY
 
 #define SINK_POWER 1
+var/global/list/moth_amount = 0 // Chompstation Addition, Rykka waz here. *pawstamp*; CitRP: It's for the moth spawn thingy
```
Comment (silicons) [inline code/modules/mob/living/simple_mob/subtypes/vore/solargrub.dm:21]: GLOBAL_VAR_INIT(moth_amount, 0)

**Comment by Captain277** [comment]
@Saisni This PR is good to go as soon as you resolve Kevin's requested change.

File: code/modules/mob/living/simple_mob/subtypes/vore/solargrub.dm (hunk 1 above)
Comment (Saisni) [reply code/modules/mob/living/simple_mob/subtypes/vore/solargrub.dm]: Ok, I don't know what you wanted me to do, but it just broke shit.

**Pull request merged**